fl4health/clients/basic_client.py

```diff
@@ -1034,24 +977,6 @@ def set_optimizer(self, config: Config) -> None:
         assert not isinstance(optimizer, dict)
         self.optimizers = {"global": optimizer}
 
-    def clone_and_freeze_model(self, model: nn.Module) -> nn.Module:
```

Comment:
This is a pretty generic function. Moved it to utils/client.py. Let me know if you disagree with the change.